cypress/e2e/spec.cy.js

```diff
@@ -1,5 +1,55 @@
-describe('empty spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
+describe('App', () => {
+  beforeEach(() => {
+    cy.intercept({url: 'https://stay-or-go-server.herokuapp.com/api/v1/disasters*'}, {fixture: 'disasters'})
   })
+
+  // beforeEach(() => {
+  //   cy.intercept('GET', 'https://stay-or-go-server.herokuapp.com/api/v1', {fixture: 'disasters'})
+  // })
+  it('should have a title', () => {
+    cy.visit('http://localhost:3002/').wait(1000)
```

Comment:
the default port for webpack is localhost:3000. Could we switch to that instead?